std_msgs/Header header
float64 ctl_force            # ctl_force reading
float64 ctl_force_variance   # ctl_force_variance reading. 0 is interpreted as unknown

